src/databricks/labs/lakebridge/cli.py

```diff
@@ -654,10 +655,15 @@ def analyze(
 ):
     """Run the Analyzer"""
     ctx = ApplicationContext(w)
-    ctx.add_user_agent_extra("cmd", "analyze")
-
-    logger.debug(f"User: {ctx.current_user}")
-    ctx.analyzer.run_analyzer(source_directory, report_file, source_tech)
+    try:
+        ctx.analyzer.run_analyzer(source_directory, report_file, source_tech)
+    finally:
+        exception_cls, _, _ = sys.exc_info()
+        if exception_cls is not None:
+            ctx.add_user_agent_extra("analyzer_error", exception_cls.__name__)
```

Comment:
I understand why we have this, but I think this is stretching the use of the user-agent beyond its intended purpose.

@sundarshankar89: I'm fine with this for now, but it's an example of something we need to move out to the proper CLI-integrated telemetry.